web/pf4/src/components/KChart.tsx

```diff
@@ -148,11 +151,12 @@ class KChart extends React.Component<KChartProps, State> {
           </Chart>
         </div>
         <ChartLegend
-          data={data.legend}
-          height={legendHeight}
-          responsive={false}
+          x={50}
+          data={legend.items}
+          height={legend.height}
           themeColor={ChartThemeColor.multi}
           width={this.state.width}
+          itemsPerRow={legend.itemsPerRow}
```

Comment:
With all current CSS and HTML featureset, it's hard to believe that this needs to be like this. Too _manual_ IMO!

Reply:
Yeah but this is all done in SVG so I think that's why it's more complicated